app/scripts/views/mixins/settings-mixin.js

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-// helper functions for views with a profile image. Meant to be mixed into views.
+// View mixin with functions with a profile image.
```

Comment:
This comment is still off. The module has lost a lot of its bang, too, now that there's only one settings page. Having a grab-bag of methods for settings-like views probably wasn't good organization to begin with, but we can handled that in a separate clean-up.